src/main/webapp/portal/main/partials/access-denied.html

```diff
@@ -0,0 +1,13 @@
+<frame-page app-header-title='Access denied' 
```

Comment:
Additionally, if frame apps want to create there own custom access-denied page, that is great. Not sure what a template would really give them. That seems more complex than just creating a partial with the `<frame-page>` directive.